src/elements/content-sharing/__tests__/ContentSharing.test.js

```diff
@@ -0,0 +1,339 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { mount } from 'enzyme';
+import { FormattedMessage } from 'react-intl';
+import API from '../../../api';
+import ErrorMask from '../../../components/error-mask/ErrorMask';
+import ContentSharing from '../ContentSharing';
+import UnifiedShareModal from '../../../features/unified-share-modal/UnifiedShareModal';
+import { DEFAULT_HOSTNAME_API, TYPE_FILE, TYPE_FOLDER } from '../../../constants';
+import { convertItemResponse, convertUserResponse } from '../../../features/unified-share-modal/utils/convertData';
+import {
+    MOCK_ITEM,
+    MOCK_ITEM_API_RESPONSE,
+    MOCK_CONVERTED_ITEM_DATA,
+    MOCK_CONVERTED_USER_DATA,
+    MOCK_SHARED_LINK,
+    MOCK_SHARED_LINK_DATA_AFTER_NORMALIZATION,
+    MOCK_USER_API_RESPONSE,
+} from '../../../features/unified-share-modal/utils/__mocks__/USMMocks';
+
+jest.mock('../../../api');
+jest.mock('../../../features/unified-share-modal/utils/convertData');
```

Comment:
It seems like the entire API and USM component modules should be mocked.

Reply:
Yup, I initially tried to do this as well. Unfortunately, I wasn't able to grab the `props` from the USM when I mocked it.